pkg/apiserver/README.md

```diff
@@ -0,0 +1,48 @@
+# APIServer Module Architecture
```

Comment:
we have some developer guide here, make sure this docs can be seen by add some reference, or we can directly put the docs there.

https://kubevela.io/docs/contributor/code-contribute#run-velaux-locally